compliance-toolkit-default.properties

```diff
@@ -191,6 +232,6 @@ springdoc.swagger-ui.tagsSorter=alpha
 springdoc.swagger-ui.operationsSorter=alpha
 
 #SBI BiometricsQualityCheckValidator
-mosip.toolkit.sbi.qualitycheck.finger.sdk.urls=[{"name": "Mock SDK","url": "http://localhost:9099/biosdk-service","healthUrl": "http://localhost:9099/biosdk-service/actuator/health", "includeInResults":false},{"name": "Mock SDK Dev Env","url": "https://api-internal.dev.mosip.net/biosdk-service","healthUrl": "https://api-internal.dev.mosip.net/biosdk-service/actuator/health", "includeInResults":true}]
-mosip.toolkit.sbi.qualitycheck.face.sdk.urls=[{"name": "Mock SDK","url": "http://localhost:9099/biosdk-service","healthUrl": "http://localhost:9099/biosdk-service/actuator/health", "includeInResults":false},{"name": "Mock SDK Dev Env","url": "https://api-internal.dev.mosip.net/biosdk-service","healthUrl": "https://api-internal.dev.mosip.net/biosdk-service/actuator/health", "includeInResults":true}]
-mosip.toolkit.sbi.qualitycheck.iris.sdk.urls=[{"name": "Mock SDK","url": "http://localhost:9099/biosdk-service","healthUrl": "http://localhost:9099/biosdk-service/actuator/health", "includeInResults":false},{"name": "Mock SDK Dev Env","url": "https://api-internal.dev.mosip.net/biosdk-service","healthUrl": "https://api-internal.dev.mosip.net/biosdk-service/actuator/health", "includeInResults":true}]
\ No newline at end of file
+mosip.toolkit.sbi.qualitycheck.finger.sdk.urls=[{"name": "BQAT SDK","url": "https://api-internal.dev1.mosip.net/bqatsdk-service","healthUrl": "https://api-internal.dev1.mosip.net/bqatsdk-service/actuator/health", "includeInResults":false}]
+mosip.toolkit.sbi.qualitycheck.face.sdk.urls=[{"name": "BQAT SDK","url": "https://api-internal.dev1.mosip.net/bqatsdk-service","healthUrl": "https://api-internal.dev1.mosip.net/bqatsdk-service/actuator/health", "includeInResults":false}]
+mosip.toolkit.sbi.qualitycheck.iris.sdk.urls=[{"name": "BQAT SDK","url": "https://api-internal.dev1.mosip.net/bqatsdk-service","healthUrl": "https://api-internal.dev1.mosip.net/bqatsdk-service/actuator/health", "includeInResults":false}]
```

Comment:
@VSIVAKALYAN 

Avoid hardcoding of domains

Reply:
@syedsalman3753 Fixed the hardcoded one please Merge the PR.
CC: @ckm007